frontend/src/services/userService.js

```diff
@@ -0,0 +1,12 @@
+import * as api from "./apis";
+import httpService from "./httpService";
+
+export function register(user) {
+  const endpoint = api.registerAPI;
+  return httpService.post(endpoint, {
```

Comment:
much better if your api value is `http://127.0.0.1:8000/api` 
so you dont create another endpoint all over again, you will just append the right end point like below

```suggestion
  return httpService.post(`${api}/register`, {
```